lib/modules/manager/helm-values/extract.ts

```diff
@@ -22,7 +22,7 @@ function getHelmDep({
   dep.replaceString = tag;
   dep.versioning = dockerVersioning;
   dep.autoReplaceStringTemplate =
-    '{{newValue}}{{#if newDigest}}@{{newDigest}}{{/if}}';
+    '"{{newValue}}"{{#if newDigest}}@{{newDigest}}{{/if}}';
```

Comment:
```suggestion
    '"{{newValue}}{{#if newDigest}}@{{newDigest}}{{/if}}"';
```
otherwise we get `"1.2.2"@sha256:xxxxxx`

please add a test with a docker digest